packages/server-core/src/networking/instance/instance.ts

```diff
@@ -74,7 +74,7 @@ export default (app: Application): void => {
     try {
       const adminScopes = (await app.service(scopePath).find({
         query: {
-          type: 'admin:admin'
+          type: 'instance:read'
```

Comment:
This might be a situation where we actually want to do something just for admins, though the distinction might not matter too much. I think what this is intended for is admins looking at /admin/instances, so that the table can remove instances that have ended. I don't think a user without admin:admin would ever have instance:read or instance:write, but in the unlikely event that happens, I don't think they'd have any need of being notified when an instance is removed.